packages/mdc-list/README.md

```diff
@@ -40,9 +40,21 @@ npm install @material/list
 
 ```html
 <ul class="mdc-list" aria-orientation="vertical">
-  <li class="mdc-list-item">Single-line item</li>
-  <li class="mdc-list-item">Single-line item</li>
-  <li class="mdc-list-item">Single-line item</li>
+  <li class="mdc-list-item">
+    <span class="mdc-list-item__text">
```

Comment:
wrap this into single line?

`<span class="mdc-list-item__text">Single-line item</span>`